### ISSUE TITLE
[Feature Suggestion] Add wallet list on homescreen (and delete wallet)

### ISSUE THREAD

**Issue by shivchawla** [trustee-wallet/trusteeWallet#78]
Currently, the list of wallets is only available in settings screen which is confusing. I think adding all the wallets on home screen would be a good feature to add. What do yo think?

Also, there is no way to delete the wallet. It will be great to add that feature as well. 

Thanks!

**Comment by Turtus** [comment]
1) we are searching for better ui - it`s planned till summer to be added with charts and portfolio features

2) delete could be dangerous for begginers - as not everyone still understands that we are not holding their mnemonics and couldnt recover accounts if not backuped - but also there is one planned task but only on fully backuped and proved to have backup wallet

**Comment by shivchawla** [comment]
Yes, I agree! Only if wallet is fully backed up, user should be allowed to delete. Also, if a wallet was added via import (by adding Pneumonic), it should be easy to delete as it means it was backed up., right?

**Comment by Turtus** [comment]
done
